test-data/unit/check-incremental.test

```diff
@@ -4884,3 +4884,60 @@ def f(x: str) -> None: pass
 [out]
 [out2]
 main:2: error: Argument 1 to "f" has incompatible type "int"; expected "str"
+
+[case testFollowImportSkipNotInvalidatedOnPresent]
+# flags: --follow-imports=skip
+# cmd: mypy -m main
+[file main.py]
+import other
+[file other.py]
+x = 1
+[file other.py.2]
+x = 'hi'
+[stale]
+[rechecked]
+
+[case testFollowImportSkipNotInvalidatedOnPresentPackage]
+# flags: --follow-imports=skip
+# cmd: mypy -m main
+[file main.py]
+import other
+[file other/__init__.py]
+x = 1
+[file other/__init__.py.2]
+x = 'hi'
+[stale]
+[rechecked]
+
+[case testFollowImportSkipNotInvalidatedOnAdded]
+# flags: --follow-imports=skip --ignore-missing-imports
+# cmd: mypy -m main
+[file main.py]
+import other
+[file other.py.2]
+x = 1
+[stale]
+[rechecked]
+
+[case testFollowImportSkipInvalidatedOnAddedStub]
+# flags: --follow-imports=skip --ignore-missing-imports
+# cmd: mypy -m main
+[file main.py]
+import other
```

Comment:
Should we have test cases that use `from pkg import foo`, where `pkg` either is a package that exists but is skipped, or gets created? `pkg.foo` is a module.

Reply:
There is already a test when `pkg` was actually missing and gets created, see `testAddedMissingStubsPackageFrom`. I will add a test for when it was just skipped.